notes/points-markets-settlement.md

```diff
@@ -98,6 +98,16 @@ In the event that the data required by the ***Valuation Algorithm*** are availab
 Note that due to the specifics of oracle protocols, etc. there may be a delay between the conditions for triggering an ***Imminent Airdrop Event*** and the data reaching a market on-chain and the cessation of trading.
 
 
+### Interim Airdrop Event
+
+An ***Interim Airdrop Event*** is deemed to have occurred when some asset with tradable value is issued to at least `Interim Inclusivity Threshold` (%) of all issued *points* (including points issued after the ***Interim Airdrop Event*** snapshot) without *conversion* of the points, i.e. accounts are still deemed to have a certain balance of points assigned to them which could receive further rewards in future.
+
+Once an ***Interim Airdrop Event*** has occurred and all input to the ***Valuation Algorithm*** is available, the *total issued market value* can be calculated. This should be calculated identically to the VWAP in the ***Valuation Algorithm*** for any final token issuance. 
```

Comment:
It's a little unclear for me from this whether I am calculating the *total issued market value* at the point of the interim airdrop event occurrence, or only later at settlement. I think the intention is at the "interim airdrop event occurance". 

I.e. to take HLP and PURR as an example: the *total issued market value* should now be getting calculated "identivally to the VWAP rules of the valuation algorithm, once this is done this number can be recorded in the settlement document on ipfs that the UMA oracle is pointing to. Is that the intention?

Reply:
That is the intention and good call on people having an update to the settlement document they can vote on to specify the number rather than just remembering it!